docs/UserGuide.md

```diff
@@ -101,44 +101,108 @@ Adds a task to the planner.
+
+* [Basic Usage](#basic-usage) (For users new to command line applications)
+* [Intermediate Usage](#intermediate-usage) (For users comfortable with command line applications)
+* [Advanced Usage](#advanced-usage) (Extension to *Intermediate Usage*)
+
+#### Basic Usage
+
+The `list` command accepts 2 parameters.
+You can tell the command to filter your tasks based on
+
+* the range of days you want to see
+* the importance of the tasks
+
+##### Command Format
+
+`list f/FORECAST i/IMPORTANCE`
+
+* `FORECAST`: **DAY** to show the tasks today, **WEEK** to show the tasks one week from now, **ALL** to show the tasks within one month from now.
+
+* `IMPORTANCE`: One of **HIGH**, **MEDIUM**, **LOW**, **ALL**.
+
+##### Example Usage
+
+The following shows the output from ATHENA after entering `list f/Week i/High`.
+You should expect to see all **high importance tasks** occuring **within one week from now**.
+
+*TODO: Insert screenshot*
+
+#### Intermediate Usage
+
+The `list` command accepts 2 parameters.
+You can tell the command to filter your tasks based on
+
+* the range of days you want to see
+* the importance of the tasks
+
+These parameters are optional. If they are left empty, ATHENA will use the default value assigned.
+
+##### Command Format
+
 `list [f/FORECAST] [i/IMPORTANCE]`
 
-#### Parameters
-* `FORECAST`: **DAY** to show the tasks today, **WEEK** to show the tasks this week, **ALL** to show all tasks. 
+* `FORECAST`: **DAY** to show the tasks today, **WEEK** to show the tasks within one week from now, **ALL** to show the tasks within one month from now.
 
   Default: **WEEK**.
-* `IMPORTANCE`: One of **HIGH**, **MEDIUM**, **LOW**, **ALL**. 
+
+* `IMPORTANCE`: One of **HIGH**, **MEDIUM**, **LOW**, **ALL**.
 
   Default: **ALL**.
 
-#### Shortcut format
-* `l3` = `list i/HIGH`
-* `l2` = `list i/MEDIUM`
-* `l1` = `list i/LOW`
-* `lw` = `list f/WEEK`
-* `ld` = `list f/DAY`
-* `la` = `list f/ALL`
-  
-#### Example usage 
+##### Example Usage
 
-* `l`
-* `ld`
-* `list f/WEEK i/medium`
-* `l2 f/WEEK`
+The following shows the output from ATHENA after entering `list f/Week i/High`.
+You should expect to see all **high importance tasks** occuring **within one week from now**.
 
-#### Expected output
+*TODO: Insert screenshot*
+
+Alternatively, you can leave ATHENA to use the default values as specified above.
+For example, entering `list i/Low` will show all **low importance** tasks occuring **within one week from now**.
+
+*TODO: Insert screenshot*
 
-When you input `l`:
+#### Advanced Usage
 
-![l command screenshot](screenshots/athena%20l.jpg)
+This section is an extension to the [Intermediate Usage](intermediate-usage) section.
 
-When you input `ld`:
+The `list` command supports shortcuts for advanced users.
+
+##### Supported Shortcuts
+
+The following commands on the left will be expanded to the corresponding commands on the right.
+
+| Shortcut | Expanded command |
+|----------|------------------|
```

Comment:
Is it missing the high importance shortcut?

Reply:
Oh yea maybe accidentally deleted that line. Nice catch